ecommerce/frontend/src/components/UserReview.less

```diff
@@ -0,0 +1,51 @@
+.user-review {
+    display: flex;
+    flex-direction: column;
+    // gap: 10px;
+    padding: 2px 2%;
+    h1 {
+        font-size: xx-large;
+    }
+}
```

Comment:
I would suggest a white background and a `border: 2px gainsboro solid` to be consistent with the product header 
![image](https://user-images.githubusercontent.com/13124263/101788564-49964600-3b11-11eb-81d9-944cdbaa6280.png)